crates/iota-sdk-transaction-builder/src/lib.rs

```diff
@@ -128,8 +128,6 @@
 //! These methods set various metadata which may be needed for the execution.
 //!
 //! - [gas](TransactionBuilder::gas): Add a gas coin to pay for the execution.
```

Comment:
Maybe we want to slightly adapt this comment?

Reply:
https://github.com/iotaledger/iota-rust-sdk/pull/335/commits/c074b81a2ecfb54c807bde452eb5a377f53279b0